net/netcheck/netcheck.go

```diff
@@ -181,6 +182,11 @@ type Client struct {
 	// If nil, portmap discovery is not done.
 	PortMapper *portmapper.Client // lazily initialized on first use
 
+	// UseDNSCache controls whether this client should use a
+	// *dnscache.Resolver to resolve host names. If false, the default
+	// net.Resolver will be used, with no caching.
```

Comment:
clarify which hostnames? If only DERP hostnames, then also clarify that DERP hostnames aren't resolved if the IP(s) are provided in the DERP map, as they typically are.

So this then is only used for people with custom DERPs who don't spell out the IP addresses?